pkg/container/network.go

```diff
@@ -116,11 +116,10 @@ func bridge(iface *net.Interface) (d *DHCPInterface, err error) {
 	tapName := constants.TAP_PREFIX + iface.Name
 	bridgeName := constants.BRIDGE_PREFIX + iface.Name
 
-	var handle *netlink.Handle
 	var bridge *netlink.Bridge
-	var tuntap, link netlink.Link
+	var eth, tuntap netlink.Link
 
-	if handle, err = netlink.NewHandle(); err != nil {
+	if eth, err = netlink.LinkByIndex(iface.Index); err != nil {
```

Comment:
nit: I prefer := on the line before errcheck and no var declaration

Reply:
This is using named returns, if I do that then it won't assign the error to the named `err` but create a new block-specific one, and that's a bug. But I'll refactor this to not use named returns, they don't benefit this function anymore.